.github/workflows/nightly_tests.yml

```diff
@@ -50,36 +46,35 @@ jobs:
           path: reports
 
   run_nightly_tests_for_torch_pipelines:
-    name: Torch Pipelines CUDA Nightly Tests
+    name: Nightly Torch Pipelines CUDA Tests
     needs: setup_torch_cuda_pipeline_matrix
     strategy:
       fail-fast: false
+      max-parallel: 8
       matrix:
         module: ${{ fromJson(needs.setup_torch_cuda_pipeline_matrix.outputs.pipeline_test_matrix) }}
     runs-on: [single-gpu, nvidia-gpu, t4, ci]
     container:
       image: diffusers/diffusers-pytorch-cuda
-      options: --shm-size "16gb" --ipc host -v /mnt/cache/.cache/huggingface/diffusers:/mnt/cache/ --gpus 0
+      options: --shm-size "16gb" --ipc host --gpus 0
     steps:
       - name: Checkout diffusers
         uses: actions/checkout@v3
         with:
           fetch-depth: 2
       - name: NVIDIA-SMI
         run: nvidia-smi
-
       - name: Install dependencies
         run: |
           python -m venv /opt/venv && export PATH="/opt/venv/bin:$PATH"
           python -m uv pip install -e [quality,test]
           python -m uv pip install accelerate@git+https://github.com/huggingface/accelerate.git
           python -m uv pip install pytest-reportlog
-
+          python -m uv pip install hf_transfer
```

Comment:
Let's have this installed in our Dockerfile installed.

Reply:
LoRA tests still run here
https://github.com/huggingface/diffusers/blob/96b0e1d644b7eda549172e114fe55965b7919c98/.github/workflows/push_tests.yml#L113

Reply:
I added installing PEFT from source as well.